debian/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:jessie
+FROM debian:stretch-slim
```

Comment:
End of an era, Jess. :disappointed: :cry:

Reply:
:broken heart:

Reply:
💔